clients/admin-ui/src/features/privacy-requests/configuration/StorageConfiguration.tsx

```diff
@@ -27,37 +27,48 @@ const StorageConfiguration = () => {
   const { successAlert } = useAlert();
   const { handleError } = useAPIHelper();
   const [storageValue, setStorageValue] = useState("");
-  const [saveStorageType, { isLoading }] = useCreateStorageMutation();
-  const [saveActiveStorage] = useCreateConfigurationSettingsMutation();
+
   const { data: storageDetails } = useGetStorageDetailsQuery({
     type: storageValue,
   });
+  const [saveStorageType, { isLoading }] = useCreateStorageMutation();
+  const [saveActiveStorage] = useCreateConfigurationSettingsMutation();
+
+  useEffect(() => {
+    if (storageDetails) {
```

Comment:
Is there an API model response for useGetStorageDetailsQuery? If so, I would have this strongly typed.